src/usr/local/www/system.php

```diff
@@ -456,7 +456,8 @@ function is_timezone($elt) {
 	'Timezone',
 	$pconfig['timezone'],
 	array_combine($timezonelist, $timezonelist)
-))->setHelp('Select the timezone or location within the timezone to be used by this system.');
+))->setHelp('Select the timezone or location within the timezone to be used by this system. '.
+	'Usually choose a "Continent/City". Only choose a special or "Etc" entry if you understand why you need to use it.');
```

Comment:
I'd flip this a bit to strengthen the point, this would change both lines:
`'Select a geographic region name (Continent/Location) to determine the timezone for the firewall. '`
and
`'<br/>Choose a special or "Etc" zone only in cases where the geographic zones do not properly handle the clock offset required for this firewall.'`

Thoughts?